libraries/AP_MSP/Tools/osdtest.parm

```diff
@@ -1,8 +1,11 @@
+OSD_TYPE         3.00
+SERIAL2_PROTOCOL 33.0
```

Comment:
the reason I didn't include the SERIAL2_PROTOCOL setting is this set of params can also be used on a real board which will likely be on a different serial port

Reply:
the wiki talks about that in the general config section, but this simplifies the SITL....

Reply:
added the note in  wiki PR #3000

Reply:
closing in light of the new --msposd option